content/en/docs/architecture/ci-operator.md

```diff
@@ -362,6 +362,33 @@ releases:
         upper: "4.5.0-0"
 {{< / highlight >}}
 
+### Testing With a Cluster from a Cluster Pool
+Instead of using `releases` stanza in the configuration, a job can *claim* a cluster from a cluster pool and use it for testing _"something on OpenShift"_.
+
+```yaml
+- as: e2e
+  cluster_claim:
+    cloud: aws
+    product: ocp
+    owner: dpp
+    version: "4.7"
+    timeout: 1h
+  container:
+    from: cli
+  commands: |
+    printenv KUBECONFIG
+    printenv KUBEADMIN_PASSWORD_FILE
+    oc whoami
+    oc config view
+    oc get node
+```
+
+The `cluster_claim` above claims an `ocp` cluster with `4.7` version on `aws`. The owner of the `aws` account with which the cluster is installed is `dpp`. If the claimed cluster is running in `1h`, the environment variable `${KUBECONFIG}` stores the path to the `admin`'s `kubeconfig` file that can be used to contacting the cluster. The `kubeadmin`'s password is stored in `${KUBEADMIN_PASSWORD_FILE}`.
```

Comment:
If I am a user, how do I know what "owner" should I set? How do I know? The document should advise. Same with `timeout`: why should I set it myself and why should I care?

Reply:
Updated.